odl/tomo/geometry/conebeam.py

```diff
@@ -362,17 +369,29 @@ def src_position(self, angle):
         array([ 0., -2.])
         >>> np.allclose(geom.src_position(np.pi / 2), [2, 0])
         True
+
+        The method is vectorized, i.e., it can be called with multiple
+        angles at once:
+
+        >>> points = geom.src_position([0, np.pi / 2])
+        >>> np.allclose(points[0], [0, -2])
+        True
+        >>> np.allclose(points[1], [2, 0])
+        True
         """
-        if angle not in self.motion_params:
-            raise ValueError('`angle` {} is not in the valid range {}'
-                             ''.format(angle, self.motion_params))
+        squeeze_out = np.isscalar(angle)
```

Comment:
Good point. I guess I would like to change this to `np.shape(angle) == ()` which covers both cases.